Source/Scene/ClassificationPrimitive.js

```diff
@@ -0,0 +1,916 @@
+     * and the appearance defines the visual characteristics.  Decoupling geometry and appearance allows us to mix
+     * and match most of them and add a new geometry or appearance independently of each other. Only the {@link PerInstanceColorAppearance}
+     * is supported at this time.
+     * </p>
+     * <p>
+     * For correct rendering, this feature requires the EXT_frag_depth WebGL extension. For hardware that do not support this extension, there
+     * will be rendering artifacts for some viewing angles.
+     * </p>
+     * <p>
+     * Valid geometries are {@link BoxGeometry}, {@link CylinderGeometry}, {@link EllipsoidGeometry}, {@link PolylineVolumeGeometry}, and {@link SphereGeometry}.
+     * </p>
+     * <p>
+     * Geometries that follow the surface of the ellipsoid, such as {@link CircleGeometry}, {@link CorridorGeometry}, {@link EllipseGeometry}, {@link PolygonGeometry}, and {@link RectangleGeometry},
+     * are also valid if they are extruded volumes; otherwise, they will not be rendered.
+     * </p>
+     *
+     * @alias ClassificationPrimitive
+     * @constructor
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {Array|GeometryInstance} [options.geometryInstances] The geometry instances to render.
+     * @param {Boolean} [options.show=true] Determines if this primitive will be shown.
+     * @param {Boolean} [options.vertexCacheOptimize=false] When <code>true</code>, geometry vertices are optimized for the pre and post-vertex-shader caches.
+     * @param {Boolean} [options.interleave=false] When <code>true</code>, geometry vertex attributes are interleaved, which can slightly improve rendering performance but increases load time.
+     * @param {Boolean} [options.compressVertices=true] When <code>true</code>, the geometry vertices are compressed, which will save memory.
+     * @param {Boolean} [options.releaseGeometryInstances=true] When <code>true</code>, the primitive does not keep a reference to the input <code>geometryInstances</code> to save memory.
+     * @param {Boolean} [options.allowPicking=true] When <code>true</code>, each geometry instance will only be pickable with {@link Scene#pick}.  When <code>false</code>, GPU memory is saved.
+     * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready. If false initializeTerrainHeights() must be called first.
+     * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Determines if this primitive's commands' bounding spheres are shown.
+     * @param {Boolean} [options.debugShowShadowVolume=false] For debugging only. Determines if the shadow volume for each geometry in the primitive is drawn. Must be <code>true</code> on
+     *                  creation for the volumes to be created before the geometry is released or options.releaseGeometryInstance must be <code>false</code>.
+     *
+     * @see Primitive
```

Comment:
Do you also want `Primitive` and `GroundPrimitive` to reference this with `@see`?